README.md

```diff
@@ -1 +1,17 @@
-# repository-template
+# Chocolatey choco-playwright
+
+**NOTE: This project is used on Chocolatey websites and is being released for the benefit of the community. While we endeavour to help and fix issues, it will be limited to GitHub issues, discussions and pull requests when we are able to.**
+
+This repository holds all common Playwright specific code that is used across many Chocolatey projects.
+
+## Commands
+
+Below are a list of commands and 
```

Comment:
This feels like it got interrupted...

Reply:
I've just removed this sentence. I don't think it is needed. I originally was going to put more stuffs in this section but moved it over to Confluence instead.